conf/landorpropertydisposal.routes

```diff
@@ -79,4 +79,7 @@ POST       /:srn/change-is-seller-connected-party/:index/:disposalIndex
 GET        /:srn/land-property-sale-independent-valuation/:landOrPropertyIndex/:disposalIndex                          controllers.nonsipp.landorpropertydisposal.DisposalIndependentValuationController.onPageLoad(srn: Srn, landOrPropertyIndex: Max5000, disposalIndex: Max50, mode: Mode = NormalMode)
 POST       /:srn/land-property-sale-independent-valuation/:landOrPropertyIndex/:disposalIndex                          controllers.nonsipp.landorpropertydisposal.DisposalIndependentValuationController.onSubmit(srn: Srn, landOrPropertyIndex: Max5000, disposalIndex: Max50, mode: Mode = NormalMode)
 GET        /:srn/change-land-property-sale-independent-valuation/:landOrPropertyIndex/:disposalIndex                   controllers.nonsipp.landorpropertydisposal.DisposalIndependentValuationController.onPageLoad(srn: Srn, landOrPropertyIndex: Max5000, disposalIndex: Max50, mode: Mode = CheckMode)
-POST       /:srn/change-land-property-sale-independent-valuation/:landOrPropertyIndex/:disposalIndex                   controllers.nonsipp.landorpropertydisposal.DisposalIndependentValuationController.onSubmit(srn: Srn, landOrPropertyIndex: Max5000, disposalIndex: Max50, mode: Mode = CheckMode)
\ No newline at end of file
+POST       /:srn/change-land-property-sale-independent-valuation/:landOrPropertyIndex/:disposalIndex                   controllers.nonsipp.landorpropertydisposal.DisposalIndependentValuationController.onSubmit(srn: Srn, landOrPropertyIndex: Max5000, disposalIndex: Max50, mode: Mode = CheckMode)
+
+GET        /:srn/land-property-disposal-list/:page                 controllers.nonsipp.landorpropertydisposal.LandOrPropertyDisposalListController.onPageLoad(srn: Srn, page: Int, mode: Mode = NormalMode)
```

Comment:
Does this need landOrPropertyIndex too?

Reply:
Nah no index required as I fetch all disposals (which returns a `Map[String, Map[String, Disposal]]`) - the 2 strings are the indexes so the code should be fairly neat